lib/utils/testhelpers.go

```diff
@@ -87,3 +91,28 @@ func RunTestBackgroundTask(ctx context.Context, t *testing.T, task *TestBackgrou
 		}
 	})
 }
+
+// RequireRoot skips the current test if it is not running as root.
+func RequireRoot(t *testing.T) {
+	t.Helper()
+	if os.Geteuid() != 0 {
+		t.Skip("This test will be skipped because tests are not being run as root.")
+	}
+}
+
+// GenerateLocalUsername generates the username for a local user that does not
+// already exist (but it does not create the user).
+func GenerateLocalUsername(t *testing.T) string {
+	login := uuid.NewString()[:12]
```

Comment:
This will have a in it dash, right?

Will this cause problems if the username begins with a number?

Should we do something more directed, maybe "teleport-$(random hex characters from crypto/rand.Reader)"